theseus/embodied/collision/tests/test_signed_distance_field.py

```diff
@@ -17,10 +17,12 @@ def test_sdf_2d_shapes():
         for field_width in [1, 10, 100]:
             for field_height in [1, 10, 100]:
                 for num_points in [1, 10, 100]:
-                    origin = torch.randn(batch_size, 2)
-                    sdf_data = torch.randn(batch_size, field_width, field_height)
-                    points = torch.randn(batch_size, 2, num_points)
-                    cell_size = torch.randn(batch_size, 1)
```

Comment:
Hmm were these tests also broken? I guess they somehow passed before because of calling .data on torch.tensor?

Reply:
Turns out we are note checking that these inputs are Variables, so one can pass a tensor by mistake. They passed before because of what you said.